int64 idx_start
int64 window_size
float64 x
float64 y
float64 z
float64 roll
float64 pitch
float64 yaw
bool publish
